.github/instructions/learning-journey/_index/frontmatter.yaml

```diff
@@ -0,0 +1,119 @@
+# - "Learn how to monitor a Kubernetes cluster in Grafana Cloud."
+# - "Welcome to the Grafana learning journey that provides the best practice for configuring Alloy to send logs to Grafana Cloud."
+description: "<JOURNEY_DESCRIPTION>"
+
+# Ordering weight for navigation (use increments of 100)
+# Orders learning journeys on the index and navigation
+weight: <WEIGHT_NUMBER>
+
+# Learning journey metadata
+journey:
+  # Skill level for the target audience
+  skill: <SKILL_LEVEL> # Options: Beginner, Intermediate, Advanced
+  # Source type for the journey content
+  source: "<CONTENT_SOURCE>" # Example: "Docs & blog posts"
+  # Journey logo configuration
+  logo:
+    # Path to logo image file
+    src: <LOGO_PATH> # Example: /static/img/menu/alloy-icon.svg
+    # Background color for the logo (hex color code)
+    background: "<BACKGROUND_COLOR>" # Example: '#0068FF'
+    # Logo width in pixels
+    width: <LOGO_WIDTH> # Example: 46
+    # Logo height in pixels
+    height: <LOGO_HEIGHT> # Example: 55
+
+# Step number (always 1 for _index.md files)
+step: 1
+
+# Layout template for learning journey pages
+layout: single-journey
+
+# Cascade configuration applies to all pages in this journey
+cascade:
+  # Layout template inherited by all journey step pages
+  layout: single-journey
+
+# Call-to-action button configuration
+# Pattern: CTAs create engagement and guide users to the next action
+# For _index.md files, always use type "start" to begin the learning journey
+# Standard values: title: "Are you ready?" and cta_text: "Let's go!" are most common
+cta:
+  # CTA type for the journey start page – always "start" for _index.md files
+  type: start
+  # CTA section title text – appears above the button
+  title: "<CTA_TITLE>" # Common: "Are you ready?"
+  # CTA button text – the clickable button text
+  cta_text: "<CTA_BUTTON_TEXT>" # Common: "Let's go!"
+
+# Optional fields – include only when needed
+
+# Shorter navigation title when the main title is too long for menus
+# Pattern: Use when the full title is over 50 characters or contains redundant words for navigation
+# Guidelines:
+# 1. Remove "learning journey" or similar meta-language
+# 2. Keep the core action verb and main product/technology
+# 3. Aim for 20-35 characters when possible
+# 4. Preserve the essential meaning while being concise
+# Examples:
+# - Full title: "Monitor Linux servers with Grafana Alloy" → menuTitle: "Monitor a Linux server"
+# - Full title: "Explore logs using Logs Drilldown" → menuTitle: "Logs Drilldown"
+# - Full title: "Use Traces Drilldown to identify high latency service requests" → menuTitle: "Identify high latency service requests"
+# - Full title: "Monitor collector health using Grafana Fleet Management" → menuTitle: "Monitor collector health"
+menuTitle: "<SHORT_MENU_TITLE>"
+
+# SEO keywords for search optimization (3-5 relevant keywords)
```

Comment:
It's debatable whether these offer SEO value, but the tagging here may be useful for other purposes. As a result, I'd suggest we revise the comment to simply "keywords for optimization (3 to 5 relevant keywords)"